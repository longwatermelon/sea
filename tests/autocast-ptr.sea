fn main() -> int {
    let arr:int* = stalloc(4,int);
    arr[0] = 2;
    arr[1] = 9;
    arr[2] = 4;
    arr[3] = 8;

    let x:int* = arr+3;
    return *x;
}
