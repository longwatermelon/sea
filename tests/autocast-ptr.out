RETURN_CODE:8
STDOUT:
